service/src/main/java/uk/gov/hmcts/reform/fpl/model/event/ManageOrdersEventData.java

```diff
@@ -16,6 +16,7 @@
 @Jacksonized
 public class ManageOrdersEventData {
     Order manageOrdersType;
+    Object manageOrdersApprovedAtHearingList;
```

Comment:
In #2663 it was shown that we could now directly store DynamicLists in case data, want to update this to do so?

Reply:
👍